README.md

```diff
@@ -1,5 +1,5 @@
 <!---
-Copyright 2023 The HuggingFace Team. All rights reserved.
+Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
It seems that you have replaced all the years in all files. This should not be done. Please revert this change. The year only needs to be updated in the files that you created for this PR.

Reply:
I'm sorry...